doc/md: documenting creation of encryption keys (#643)
ariga/atlas: doc/md/deployment/intro.md

4. ```5800:5800 arigaio/atlas:latest``` use the latest atlas image, for a versioned release use a specific tag such as ```arigaio/atlas:0.3.6```.
5. ```serve --addr :5800 --storage "mysql://root:pass@tcp(host.docker.internal:3306)/atlas"``` serve Atlas on port 5800 with a persistent MySQL Database.  

### Encryption

Atlas uses [tink](https://github.com/google/tink), a battle-tested encryption library created at Google, to 

> encrypt all sensitive information.  Following recommendations from the developers of tink, Atlas

Reviewer: ```suggestion
encrypt all sensitive information. Following recommendations from the developers of tink, Atlas
```

uses [AEAD](https://developers.google.com/tink/aead?hl=en) encryption with an AES256_GCM type key.

> On its first run, Atlas will generate a keyset.json file under `$HOME/.atlas/`  containing 

Reviewer 2: ```suggestion
On its first run, Atlas will generate a keyset.json file under `$HOME/.atlas/` containing 
```

> On its first run, Atlas will generate a keyset.json file under `$HOME/.atlas/`  containing 

Reviewer: ```suggestion
On its first run, Atlas generates a keyset.json file under `$HOME/.atlas/`  containing 
```
